src/components/ComingSoon/ComingSoon.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <section>
+    <Heading class="title" size="large" weight="medium">{{ title }}</Heading>
+    <div class="text-info">
+      <TextParagraph class="description" size="medium"> {{ description }}</TextParagraph>
+    </div>
+    <div class="image-wrapper">
+      <img :src="image" alt="Sheep working" width="360" height="264">
```

Comment:
Please remove this line in favor of using the `ComingSoonImage` component. For setting its size: you can use a normal CSS `class`; let me know if you encounter any difficulty with that and I'll be happy to help.

```js
<div class="image-wrapper">
  <ComingSoonImage class="coming-soon-image" />
</div>
```